docs/user/reference/environment-variables.rst

```diff
@@ -56,6 +56,31 @@ All :doc:`build processes </builds>` have the following environment variables au
     :Example: ``https://docs.readthedocs.io/ja/stable/``
     :Example: ``https://example--17.org.readthedocs.build/fr/17/``
 
+.. envvar:: READTHEDOCS_REPOSITORY_CLONE_URL
```

Comment:
Is there a reason we're calling it CLONE_URL, instead of REPOSITORY_URL to match the help_text? 

> This is the same URL defined in your Project's :term:`dashboard` in :menuselection:`Admin --> Settings --> Repository URL`.

Reply:
It's because the repository will also have a "web URL", i.e. the base URL for where the file can be viewed and edited, so `ssh://git@webserver.com/repo` is different from `https://frontend.com/repo/`

Reply:
Seems the doc is not correct then when it says it's the same as your Repository URL in the dashboard?

Reply:
"Repository URL in the dashboard" is what we use as `Project.repo`, which is the same of the content of this variable. Do you have a suggestion how to make this clearer? I think having all these URLs is just confusing, but necessary, tho.

I want to make the distinction between `HTML_URL` and `CLONE_URL` mainly because they are not going to be _always_ the same. In community, they will mostly be the same, tho.